jib-cli/src/test/java/com/google/cloud/tools/jib/cli/buildfile/ArchiveLayerSpecTest.java

```diff
@@ -46,26 +46,78 @@ public void testArchiveLayerSpec_nameRequired() {
     String data = "archive: out/archive";
 
     try {
-      archiveLayerSpecMapper.readValue(data, ArchiveLayerSpec.class);
+      mapper.readValue(data, ArchiveLayerSpec.class);
       Assert.fail();
     } catch (JsonProcessingException jpe) {
       MatcherAssert.assertThat(
           jpe.getMessage(), CoreMatchers.startsWith("Missing required creator property 'name'"));
     }
   }
 
+  @Test
+  public void testArchiveLayerSpec_nameNonNull() {
+    String data = "name: null\n" + "archive: out/archive";
+
+    try {
+      mapper.readValue(data, ArchiveLayerSpec.class);
+      Assert.fail();
+    } catch (JsonProcessingException jpe) {
```

Comment:
Interesting. So the mapper catches runtime exceptions and wraps it with `JsonProcessingException` after attaching the underlying runtime exception message?

Reply:
Yes, any exception is always wrapped in JsonProcessingException.